client/apollo/js/main.js

```diff
@@ -931,12 +931,7 @@ return declare( [JBPlugin, HelpMixin,Evented],
                     locationBox.set('value',locationVal,false);
                 }
                 else{
-                    var locationBoxString = Util.assembleLocStringWithLength( currRegion );
-                    if(currRegion.ref.startsWith("{")){
-                        sequenceString = locationBoxString.substring(0,locationBoxString.lastIndexOf("}")+1);
-                        sequenceObject = JSON.parse(sequenceString).sequenceList[0];
-                        locationBoxString = sequenceObject.name + locationBoxString.substr(locationBoxString.lastIndexOf(":"));
-                    }
+                    var locationBoxString = Util.assembleProjectedString(currRegion);
```

Comment:
@nathandunn I still get this error after pulling from the projection_v2_server branch.

Here is what I have in my config:
```
jbrowse {
   git {
       url= "https://github.com/GMOD/jbrowse"
       branch = "projection_v2_server"
       alwaysPull = true
       alwaysRecheck = true
   }
```